### PR TITLE
changelog for 0.10.0

### PR DESCRIPTION
**Pull request by indirectlylit**
### Summary

Add list of changes for 0.10.0

### Reviewer guidance

Make sure it's not missing anything critical and has no typos

### References

https://github.com/learningequality/kolibri/issues?q=milestone%3A0.10.0

----

### Contributor Checklist

- [x] Contributor has fully tested the PR manually
- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -3,6 +3,24 @@ Release Notes
...
+ - Content import/export is more reliable and easier to use
+ - Search has improved results and handles duplicate items
+ - Display of answer history in learner exercises is improved
+ - Login page is more responsive
+ - Windows-specific improvements and bug fixes
+ - New Kolibri configuration file
+ - Overall improved performance
+ - Auto-play videos
+ - Command to migrate content directory location
+
```
Comment (christianmemije) [inline CHANGELOG.rst]: No mention of the pdf renderer improvements :broken_heart:

Comment (indirectlylit) [reply CHANGELOG.rst]: on it!

Comment (indirectlylit) [reply CHANGELOG.rst]: remember to use this!
image

Comment (indirectlylit) [reply CHANGELOG.rst]: updated

**Pull request merged**